Dockerfile

```diff
@@ -1,22 +1,43 @@
-FROM kbase/sdkbase2 as build
+FROM eclipse-temurin:11-jdk as build
```

Comment:
The official openjdk images have been deprecated, with a list of recommendations for alternatives given. Eclipse seemed as good as any to choose.

Other possible build images:
- amazoncorretto (runs on Amazon linux)
- ibmjava
- ibm-semeru-runtimes
- kbase/sdkbase2 (bleh, would need java 11 installing)

Reply:
We're running GHA tests with temurin so this is a good choice